README.md

```diff
@@ -97,18 +97,21 @@ Example `timeseries` request
 }
 ```
 
+There is no relation between `target` in request and response. For one requested `target`, you can return multiple `targets` in response.
+
 Example `timeseries` response
+
 ``` javascript
 [
   {
-    "target":"upper_75", // The field being queried for
+    "target":"returned target1",
```

Comment:
Shouldn‘t we say the query target?

Reply:
The thing is it's not query target or related at all 🤔

Reply:
> The thing is it's not query target or related at all 🤔

I understand technically. Yet, it doesn't seem to make sense to query for apple harvest and get cow population returned?

Shouldn't we say something like:

> Response targets can, but must not necessarily, match query targets. Is is valid to return multiple response targets per query target.

That indicates the possibility but doesn't encourage it as default or sensible behaviour?

Reply:
Btw. for my datasource I get queries by UUID and return targets by name so that it looks nicer, yet still 1:1. Maybe we could add something like this as an example?

Reply:
My use case is that I put the Query name into `target`. The Query is than constructed on backend and sent to querybus. It returns 1-n targets. Eg. `FindGlobalTraffic` returns `in`, `out` and `95pctl` targets. The usecase with this DS is very flexible :D 

I tried to abstract from usecase or concrecte example as much as possible. Especially to avoid things like `Response targets can`. As there is no relation between it, I wanted to disconnect it completely. To say "send X but receive whatever you want".

To me it seems the most reasonable to return name or description of series in "target".